WatchDogEclipsePlugin/WatchDog/src/nl/tudelft/watchdog/ui/util/UIUtils.java

```diff
@@ -274,4 +275,14 @@ public static Link createLinkedLabel(Composite parent,
 		return link;
 	}
 
+	/** Creates a Combo List of String items. */
```

Comment:
I think there is one other Combo box used in WatchDog in UserRegistrationPage.java. Could you make that use your new UIUtils function? :)